coco/endpoint.py

```diff
@@ -6,7 +6,7 @@
 from copy import copy
 from . import Result
 
-logger = logging.getLogger("asyncio")
+logger = logging.getLogger(__name__)
```

Comment:
I think all asyncio functions share the `"asyncio"`-logger. Are you sure this works?

Reply:
I continue to be confused by python logging, but my understanding is that loggers inherit the configuration of their parent loggers, where parent is

> The name is potentially a period-separated hierarchical value, like `foo.bar.baz` (though it could also be just plain `foo`, for example). Loggers that are further down in the hierarchical list are children of loggers higher up in the list.

By using `__name__` the formatting set up in `__init__.py` gets applied to this handler and the logging level set on the root logger in `master.py` gets used. Otherwise we would have to configure this logger instance at some point.

With this configuration I see logs from all the modules at the log level set in the config.

Reply:
But do you see log lines from the async functions?

Reply:
no

Reply:
I mean I do for those in the modules that are included in the logger

Reply:
e.g.
`[2019-07-03 10:34:48 -0700] [10932] [INFO] [coco.scheduler] Skipping scheduled endpoint status because cluster/running doesn't exist.
[2019-07-03 10:34:58 -0700] [10932] [INFO] [coco.scheduler] Skipping scheduled endpoint status because cluster/running doesn't exist.`

Reply:
I don't see any logs from the `asyncio` logger